### PR TITLE
Sign early ts setalgebra commits

### PR DESCRIPTION
**Pull request by TimSheard**
This branch signs the early commits fro ts-setalgebra

### CODE REVIEW

File: shelley/chain-and-ledger/executable-spec/bench/Main.hs [hunk 1]
```diff
@@ -67,13 +92,17 @@ profileCreateRegPools size = do
...
 -- =================================================
 -- Some things we might want to profile.
 
+-- main :: IO()
 -- main = profileUTxO
 -- main = includes_init_SpendOneUTxO
+-- main:: IO ()
 -- main = profileCreateRegPools 10000
 -- main = profileCreateRegPools 100000
+-- main = profileNkeysMPools
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/executable-spec/bench/Main.hs:104]: any reason to keep these comments?

File: shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Generator/Delegation.hs [hunk 2]
```diff
@@ -355,11 +355,11 @@ genGenesisDelegation coreNodes delegateKeys dpState =
           CoreKeyCred [gkey]
         )
     (GenDelegs genDelegs_) = _genDelegs $ _dstate dpState
-    genesisDelegator k = k ∈ dom genDelegs_
+    genesisDelegator k = eval(k ∈ dom genDelegs_ )
     genesisDelegators = filter (genesisDelegator . hashVKey) (fst <$> coreNodes)
-    notActiveDelegatee k = coerceKeyRole k ∉ fmap genDelegKeyHash (Map.elems genDelegs_)
+    notActiveDelegatee k = not(coerceKeyRole k `List.elem` fmap genDelegKeyHash (Map.elems genDelegs_))
```
Comment (JaredCorduan) [inline shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Generator/Delegation.hs]: why did `∉` not work?

**Review by JaredCorduan — APPROVED** [review]
Looks good to me! Very exciting!

I looks like you've addressed all @nc6's comments from #1647 (except for maybe a bit of dead code in `bench/Main.hs`. thanks!

I ran ormolu on the PR, and it moved comments inside of imports lists to the top of the file, so I removed the comments (since they would no longer be helpful).

**Comment by nc6** [comment]
Hmm, I would still prefer to have this split out much more, rather than just one giant "Internal" module. But I guess that could wait for a second PR if we want to get this in before the code freeze.

**Pull request merged**